keyboards/qwertykeys/qk100/ld/STM32F103x8_0x08006000_bootloader.ld

```diff
@@ -0,0 +1,89 @@
+/*
+    ChibiOS - Copyright (C) 2006..2016 Giovanni Di Sirio
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+/*
+ * STM32F103x8 memory setup for use with the STM32Duino bootloader.
+ */
+f103_flash_size = 128k;
+
+MEMORY
+{
+    /*Here we give 22K flash to bootloader*/
+    /*0x08005800 : the application bin start address*/
+    flash0  : org = 0x08006000, len = f103_flash_size - 0x6000
```

Comment:
This is unusual config, and deviates from the normal stm32duino bootloader config.  Is there a specific reason for this?